tensorboard/plugins/projector/projector_api_test.py

```diff
@@ -21,37 +21,66 @@
 import os
 import shutil
 
+import six
 import tensorflow as tf
 
 from google.protobuf import text_format
 
 from tensorboard.plugins import projector
 from tensorboard.util import test_util
 
-tf.compat.v1.disable_v2_behavior()
-
+def create_dummy_config():
+  return projector.ProjectorConfig(
+      model_checkpoint_path = 'test',
```

Comment:
nit: No spaces around `kwarg=value`s in Python.

Reply:
I got lazy and expected formatter to kick in :( I am spoiled!